docs/content/en/blog/releases/v0.41.0.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Release v0.41.0"
+linkTitle: "Release v0.41.0"
+date: 2022-12-20
+description: >
+ Release v0.41.0
+---
+
+## Changes since v0.40.1
+
+### Breaking Changes
+
+All __applications configuration__ file should NOT use the `spec.triggerPaths` (marked as deprecated previously), but use `spec.trigger.onCommit.paths` instead.
+
+* Remove trigger paths ([#4037](https://github.com/pipe-cd/pipecd/pull/4037))
+
+### New features
+
+From now, you can manage your project's deployment via PipeCD deployment metrics. Please refer to the [docs](/docs/user-guide/insights/).
+
+![image](/images/insights.png)
+
+* Introduce Insights feature ([#4071](https://github.com/pipe-cd/pipecd/pull/4071))
+* Support setting kubectlVersion via piped config ([#4081](https://github.com/pipe-cd/pipecd/pull/4081))
+* Add `spec.trigger.onCommit.ignores` to application config ([#4059](https://github.com/pipe-cd/pipecd/pull/4059))
```

Comment:
@knanao PlanPreview and Health Status for Cloud Run application should be added here too, right?

Reply:
Right, especially Planpreview is, but regarding health status whichever it's fine for me!